app/src/main/java/com/infomaniak/mail/utils/Exceptions.kt

```diff
@@ -17,10 +17,18 @@
  */
 package com.infomaniak.mail.utils
 
+import com.infomaniak.lib.core.api.ApiController
+import com.infomaniak.lib.core.models.ApiResponse
+import kotlinx.serialization.decodeFromString
+
 class ApiErrorException(override val message: String?) : Exception() {
 
+    inline val apiResponse get() = ApiController.json.decodeFromString<ApiResponse<Any>>(message!!)
+
     object ErrorCodes {
-        const val FOLDER_DOES_NOT_EXIST = "folder__not_exists"
+        const val DRAFT_DOES_NOT_EXIST = "draft__not_found"
+        const val DRAFT_HAS_MANY_RECIPIENTS = "draft__to_many_recipients"
```

Comment:
It doesn't sound like an error like this, I'd prefer `DRAFT_HAS_TOO_MANY_RECIPIENTS`